ARExperiment/ARExperiment/ARHelpers.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+import SceneKit
+
+extension SCNGeometry {
+    func styleFirstMaterial(with material: Any) {
```

Comment:
But then this can only be used with colors, we use it a lot with images too and other type of materials. I can rename it to `styleFirstMaterialDiffuse` though.